commons-test/src/main/java/org/infinispan/commons/test/TestNGTestListener.java

```diff
@@ -82,6 +84,7 @@ public void onStart(ISuite suite) {
 
    @Override
    public void onFinish(ISuite suite) {
+      ThreadLeakChecker.checkForLeaks(null);
```

Comment:
I know this is testing code, but what about creating another test called like checkForLeaksOnFinish() or something like that, instead of passing null and doing some logic inside the checkForLeaks?

Reply:
I looked into this and it turns out the `testName` parameter isn't needed at all -- it was only necessary before to call `testFinished(testName)`, and we don't do that any more.

Reply:
good ! 💃